### ISSUE TITLE
User can specify courses using one "batch" file

### ISSUE THREAD

**Issue by sacherus** [coursera-dl/coursera-dl#519]
### Subject of the issue
I have file with multiple slugs (500) and I want to download them
$ head ../cs3.csv
db
automata
intrologic
compilers
security
(...)

### Your environment
* Operating System (name/version): ubuntu 14.04
* Python version: 2.7
* coursera-dl version: coursera_dl version 0.6.0

### Steps to reproduce
coursera-dl -n ~/.netrc --resume --video-resolution 720p -sl en -pl < ../cs3.csv 

### Expected behaviour
User should be able to specify batch file
coursera-dl -n ~/.netrc --resume --video-resolution 720p -sl en -pl --batch-file ../cs3.csv or
coursera-dl -n ~/.netrc --resume --video-resolution 720p -sl en -pl < ../cs3.csv 

### Actual behaviour
User cannot specify batch file with multiple files:
coursera-dl -n ~/.netrc --resume --video-resolution 720p -sl en -pl < ../cs3.csv

**Comment by balta2ar** [comment]
There are numerous ways to workaround it in Linux:

```
# Use backticks
coursera-dl -n ~/.netrc --resume --video-resolution 720p -sl en -pl `cat ../cs3.csv`

# Use xargs
cat ../cs3.csv | xargs coursera-dl -n ~/.netrc --resume --video-resolution 720p -sl en -pl

# Use while read
cat ../cs3.csv | while read course; do
    coursera-dl -n ~/.netrc --resume --video-resolution 720p -sl en -pl $course
done
```

**Comment by rbrito** [comment]
I closed this issue, but I guess that it is better to have all wontfix issues open, so that people see them right away.

This sucks, but I guess that we need some kind of communication mechanism (like: "please, search previous issues to see if your problem was already reported").

Perhaps one way to get people involved in the project would be to document (say, in the wiki) a distilled version of past problems, to serve as a better knowledge base...

Opinions? Am I on crack? :grinning:

**Comment by balta2ar** [comment]
> better to have all wontfix issues open

Agree! Another reason why I left it open is that 1. other contributors may have different opinions so I'm open to discussions 2. I have to say that having such option in `youtube-dl` and `aria2c` is very convenient so maybe I will reconsider.
